cirq-core/cirq/contrib/svg/svg.py

```diff
@@ -23,6 +23,10 @@ def fixup_text(text: str):
     if '[cirq.VirtualTag()]' in text:
         # https://github.com/quantumlib/Cirq/issues/2905
         return text.replace('[cirq.VirtualTag()]', '')
+    if '<' in text:
+        return text.replace('<', '&lt;')
+    if '>' in text:
+        return text.replace('>', '&gt;')
```

Comment:
This of course raises the question whether we've got them all now :-)

Reply:
I agree. There are other open issues that track improvements to svg drawing utilities; like https://github.com/quantumlib/Cirq/issues/4499 and https://github.com/quantumlib/Cirq/issues/2313

I'd say that covering all cases is out of scope for this PR. I ran into this specific case myself and the fix added seems to be enough to unblock me.